haproxy_exporter.go

```diff
@@ -112,6 +112,7 @@ func NewExporter(uri string, haProxyServerMetricFields string, timeout time.Dura
 		18: newServerMetric("weight", "Current weight of the server.", nil),
 		33: newServerMetric("current_session_rate", "Current number of sessions per second over last elapsed second.", nil),
 		35: newServerMetric("max_session_rate", "Maximum number of sessions per second.", nil),
+		38: newServerMetric("check_duration", "Previously run health check duration, in milliseconds", nil),
```

Comment:
Following our naming guide lines http://prometheus.io/docs/practices/naming/, the metric name should include the time unit: `check_duration_milliseconds`.